Arista/ConfigFiles/telegraf-apache.conf

```diff
@@ -0,0 +1,48 @@
+###############################################################################
```

Comment:
Since we run apache on all our servers its better we move the input plugin to telegraf-default.con and update the output plugin in telegraf-linux.conf by appending "apache" to namepass.